tests/compute_engine/test_credentials.py

```diff
@@ -106,6 +106,7 @@ def test_refresh_error(self, get):
         assert excinfo.match(r"http error")
 
     @mock.patch("google.auth.compute_engine._metadata.get", autospec=True)
+    @pytest.mark.asyncio
```

Comment:
I think you would need a separate test for async here?

Reply:
Resolved